DEVELOPMENT.md

```diff
@@ -93,25 +119,66 @@ After which, you can rerun a `px auth login` to authenticate with the dev cloud
 Make sure to `px delete --clobber` if running a prior instance of Vizier pointing to another cloud instance.
 
 ## Vizier
+Vizier is Pixie’s data collector that runs on each cluster. It is responsible
+for query execution and managing PEMs.
+
+### Getting Started
+Verify that you can build Pixie Vizier and run the unit tests.
+
+The following will build Pixie Vizier and then run the unit tests for the UDF
+(User Defined Function) math operations. These UDFs will be available directly off
+the px object inside a PxL script. If you are running the development environment via
+Docker, the following should be run inside the Docker container.
+
+```
+bazel test //src/carnot/funcs/builtins:math_ops_test --test_output=errors -j 2
+```
+
+### Deploying
+Deploying a development version of Vizier is a 2-step process. An official
+release-version of Vizier must first be deployed (through the Pixie CLI or
+YAMLs) and Skaffold can then be run to build and deploy a local development
+version of Vizier.
 
-Deploying a development version of Vizier is a 2-step process. A release-version of Vizier must first be deployed (through the CLI or YAMLs), which will set up specific cluster-secrets that are not deployed via Skaffold. Skaffold can then be run to build and deploy a version of Vizier based on local changes.
+1. Since both of these steps require a Kubernetes cluster to deploy to, you must have
+Minikube installed and running. Follow the instructions
+[here](https://docs.px.dev/installing-pixie/setting-up-k8s/minikube-setup/). Note that
+both deployment scripts below use the output of `kubectl config current-context` to
+determine which Kubernetes cluster to deploy to. So make sure if you have multiple
+clusters, the context is pointing to the correct target cluster.
```

Comment:
Technically running a development version of Pixie CLoud requires a Kubernetes cluster as well, so this also applies to the Pixie Cloud section -- can we move it up?